Library/Formula/ansible-cmdb.rb

```diff
@@ -0,0 +1,26 @@
+class AnsibleCmdb < Formula
+  desc "Generates static HTML overview page from Ansible facts"
+  homepage "https://github.com/fboender/ansible-cmdb"
+  url "https://github.com/fboender/ansible-cmdb/releases/download/1.6/ansible-cmdb-1.6.zip"
+  sha256 "a201207d1a8b8929f07bd28be0ca3f1224e9ecf296756b73d938b6a41bdde80e"
+
+  depends_on :python if MacOS.version <= :snow_leopard
+  depends_on "libyaml"
+
+  def install
+    Dir.mkdir "#{bin}"
+    inreplace "Makefile", "/usr/local/lib/${PROG}", prefix
+    inreplace "Makefile", "/usr/local/bin", bin
+    inreplace "ansible-cmdb", "(bin_dir, '..', 'lib', 'ansible-cmdb')", "(bin_dir, '..')"
+    system "make", "install"
+  end
+
+  test do
+    (testpath/"hosts").write( \
```

Comment:
Can you try to use `.write <<-EOS.undent` style syntax here? The `command(` multiple line syntax can cause big problems on older Ruby versions we still attempt to support.